lib/ts/recipe/session/recipe.ts

```diff
@@ -63,9 +63,11 @@ export default class SessionRecipe extends RecipeModule {
     constructor(recipeId: string, appInfo: NormalisedAppinfo, isInServerlessEnv: boolean, config?: TypeInput) {
         super(recipeId, appInfo);
         this.config = validateAndNormaliseUserInput(this, appInfo, config);
-        logDebugMessage("session init: antiCsrf: " + this.config.antiCsrf);
+        logDebugMessage("session init: antiCsrf: " + this.config.antiCsrfFunctionOrString);
```

Comment:
you should not do this style of printing cause antiCsrfFunctionOrString can be a function.. check for that here and then print it if it's a string. Else print that it's a function.